.idea/src/java/DAO/impl/JDBCArticleDAO.java

```diff
@@ -0,0 +1,104 @@
+package DAO.impl;
+
+import DAO.ArticleDAO;
+import entity.Article;
+import entity.Author;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JDBCArticleDAO implements ArticleDAO {
+
+    private static ConnectionFactory connectionFactory;
+
+    public JDBCArticleDAO(ConnectionFactory connectionFactory) throws SQLException {
+
+        this.connectionFactory = connectionFactory;
+        createTable();
+    }
+
+    private static void createTable() throws SQLException {
+
+        String createTable = "CREATE TABLE article(article_id int PRIMARY KEY AUTO_INCREMENT, "
```

Comment:
Better to create table outside java code. Because for existing database, not in memory, this will not work